src/bigquery.rs

```diff
@@ -434,7 +438,7 @@ mod tests {
                 {"name": "test-atom", "type": "INT64", "mode": "REQUIRED"},
                 {"name": "test-null", "type": "STRING", "mode": "NULLABLE"},
                 {"name": "test-object", "type": "RECORD", "mode": "REQUIRED", "fields": [
-                    {"name": "test-nested-atom", "type": "FLOAT64", "mode": "REQUIRED"},
+                    {"name": "test-nested-atom", "type": "FLOAT64", "mode": "NULLABLE"},
```

Comment:
For context, this was a small typo that was fixed after enforcing the nullability rules.